src/instruments/src/EFB/Settings/Settings.tsx

```diff
@@ -175,6 +175,7 @@ const SimOptionsPage = () => {
     const [adirsAlignTime, setAdirsAlignTime] = usePersistentProperty('CONFIG_ALIGN_TIME', 'REAL');
     const [, setAdirsAlignTimeSimVar] = useSimVar('L:A32NX_CONFIG_ADIRS_IR_ALIGN_TIME', 'Enum', Number.MAX_SAFE_INTEGER);
     const [dmcSelfTestTime, setDmcSelfTestTime] = usePersistentProperty('CONFIG_SELF_TEST_TIME', '12');
+    const [boardingRate, setBoardingRate] = usePersistentProperty('CONFIG_BOARDING_RATE', '1');
```

Comment:
usePersistentNumberProperty might be better?

Reply:
I did take a look, NumberProperty seemed to be more useful in case of toggles and sliders, most of our current option based implementations use the regular ``usePersistentProperty``

Reply:
> most of our current option based implementations use the regular `usePersistentProperty`

Yeah, the number option is a new API. You can use an enum rather than a magic number if you use it. ;)

Reply:
I see what you mean. I'll try and change it in a while 👍🏻

Reply:
maybe I should just use the strings directly instead of using numbers in place of strings, and since it's not documented anywhere on Boarding.js file, it would naturally get confusing